adhocteam/Head-Start-TTADP
# Debug intermittent rendering issue

## Description of change
This PR started as a way to debug the intermittent rendering issues in sandbox and dev by rolling back minor changes to the regional select button, however it ended up being a rollback of the addition of touchpoints survey. The plan is to move forward with the overview widget and then debug the interaction between react-select and the survey.


## How to test
- go to sandbox
https://tta-smarthub-sandbox.app.cloud.gov/activity-reports
- refresh the landing page several times
- verify that the rending of the regional select works
- start a new report
- refresh several times
- verify that the react-select input boxes render ok

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-143


## Checklists

### Every PR


- [ n/a] API Documentation updated
- [ n/a] Boundary diagram updated
- [ n/a] Logical Data Model updated
- [n/a ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy

- [ n/a] Staging smoke test completed

### After merge/deploy
